init/kibana.conf

```diff
@@ -16,7 +16,6 @@ end script
 
 post-start script
   KB_HOME=/usr/local/kibana-4.1.4-linux-x64/
-  sh ${KB_HOME}/setDefaultIndex.sh
-  sh ${KB_HOME}/loadAssets.sh
-  sh ${KB_HOME}/refreshKibanaIndex.sh
+  /usr/bin/python ${KB_HOME}/setDefaultIndex.py
```

Comment:
Should we create a scripts directory where they are stored? i.e. the python files?